### PR TITLE
fix  drop checkpoint failure

### PR DESCRIPTION
**Pull request by panda-sheep**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Description:
fix drop checkpoint failure

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

File: src/storage/admin/DropCheckpointProcessor.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ void DropCheckpointProcessor::process(const cpp2::DropCPRequest& req) {
   auto spaceIdList = req.get_space_ids();
   auto& name = req.get_name();
   for (auto spaceId : spaceIdList) {
-    auto code = env_->kvstore_->dropCheckpoint(spaceId, std::move(name));
```
Comment (critical27) [inline src/storage/admin/DropCheckpointProcessor.cpp:16]: Em, IIRC, in the old code, the second parameter in dropCheckpoint is copied not moved from the `name`. Are you sure this PR fix some bugs?

**Comment by critical27** [comment]
Don't merge~

File: src/storage/admin/DropCheckpointProcessor.cpp (hunk 1 above)
Comment (panda-sheep) [reply src/storage/admin/DropCheckpointProcessor.cpp:16]: I will test it

Comment (panda-sheep) [reply src/storage/admin/DropCheckpointProcessor.cpp:16]: My fault

First `name` is a const reference
Secondly, the second parameter of `dropCheckpoint` is also a const reference.
move is meaningless here.

**Pull request closed without merging**